lib/ts/recipe/userroles/recipe.ts

```diff
@@ -55,6 +56,51 @@ export default class Recipe extends RecipeModule {
             if (!this.config.skipAddingPermissionsToAccessToken) {
                 SessionRecipe.getInstanceOrThrowError().addClaimFromOtherRecipe(PermissionClaim);
             }
+
+            OAuth2Recipe.getInstanceOrThrowError().addUserInfoBuilderFromOtherRecipe(
+                async (user, _accessTokenPayload, scopes, tenantId, userContext) => {
+                    let userInfo: {
+                        roles?: string[];
+                        permissons?: string[];
+                    } = {};
+
+                    if (scopes.includes("roles")) {
```

Comment:
the `permissions` scope should require the `roles` scope to be effective.

Reply:
Isn't that already the case? 

The ` if (scopes.includes("permissions")) {` block is inside the `if (scopes.includes("roles")) {` block

Reply:
oh I mistyped, it should *not* require it, the two can be used independently.

Reply:
Oh okay. I have fixed it.